packages/react/src/ImageSlider/ImageSlider.tsx

```diff
@@ -5,40 +5,57 @@
 
 import { ChevronLeftIcon, ChevronRightIcon } from '@amsterdam/design-system-react-icons'
 import clsx from 'clsx'
-import { Children, forwardRef, useEffect, useRef, useState } from 'react'
-import type { ForwardedRef, HTMLAttributes, KeyboardEventHandler, PropsWithChildren } from 'react'
+import { forwardRef, KeyboardEvent as ReactKeyboardEvent, useEffect, useRef, useState } from 'react'
+import type { ForwardedRef, HTMLAttributes } from 'react'
 import { ImageSliderContext } from './ImageSliderContext'
 import { ImageSliderItem } from './ImageSliderItem'
 import { ImageSliderScroller } from './ImageSliderScroller'
 import { ImageSliderThumbnails } from './ImageSliderThumbnails'
+import { Ratio } from '../AspectRatio'
 import { IconButton } from '../IconButton'
+import { Image } from '../Image/Image'
+
+export type SlideProps = {
+  src: string
```

Comment:
All these props are passed into `Image` and `AspectRatio` components. We should reuse their types to prevent future changes introducing issues here, e.g. `src: ImageProps['src']`.